src/redux/Signup/API.js

```diff
@@ -0,0 +1,23 @@
+import { createUserAction } from './signupReducer';
+
+const baseURL = `${process.env.REACT_APP_DOCTOR_APPOINTMENT_API_URL}/signup`;
```

Comment:
Please add a link to your API instead of env, which will facilitate our collaboration.